openverse_catalog/dags/data_refresh/refresh_view_data_task_factory.py

```diff
@@ -25,7 +26,8 @@ def create_refresh_view_data_task(data_refresh: DataRefresh):
     The task refreshes the materialized view for the given media type. The view collates
     popularity data for each record. Refreshing has the effect of adding popularity data
     for records that were ingested since the last time the view was refreshed, and
-    updating popularity data for existing records.
+    updating popularity data for existing records. It also creates a reporting task
+    which will report the status of the matview refresh once it is complete.
```

Comment:
💯 Should we add a similar note to the docs for `refresh_popularity_metrics_task_factory`?